src/libsyntax/ext/deriving/cmp/eq.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
```

Comment:
The changes to this file are essentially all spurious, right? (i.e. just from git not realising that `totaleq.rs` was renamed to overwrite `eq.rs`?)